.travis.yml

```diff
@@ -31,15 +31,19 @@ matrix:
   include:
     - env: ['os_image=ubuntu:16.04', gcc_version=5]
       services: [docker]
+    - os: osx
 
 addons:
   apt:
     packages: python2.7
 
 script:
-  # We don't want to write the script inline because of Travis kludginess --
-  # it looks like it escapes " and \ in scripts when using `matrix:`.
-  - ./build/fbcode_builder/travis_docker_build.sh
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]];
+    then
+      folly/build/bootstrap-osx-homebrew.sh;
```

Comment:
I think we should pass `-DBUILD_TESTS=ON` to this script so we build and run the tests. By default, the script does not build/run tests.